src/Network/Network/help/Az.Network.md

```diff
@@ -464,6 +464,9 @@ Gets a virtual network tap
 ### [Get-AzVirtualNetworkUsageList](Get-AzVirtualNetworkUsageList.md)
 Gets virtual network current usage.
 
+### [Get-AzVirtualNetworkGatewayVpnclientConnectionHealth](Get-AzVirtualNetworkGatewayVpnclientConnectionHealth.md)
```

Comment:
It looks like documentation wasn't updated from when the name became Get-AzVirtualNetworkGatewayVpn**C**lientConnectionHealth. Could you regenerate it?